AutofillCredentialProvider/CredentialProvider/CredentialProviderList/CredentialProviderListItemTableViewCell.swift

```diff
@@ -118,67 +143,9 @@ class CredentialProviderListItemTableViewCell: UITableViewCell {
         
         separatorInset = UIEdgeInsets(top: 0, left: contentView.layoutMargins.left + textStackView.frame.origin.x, bottom: 0, right: 0)
     }
-    
-    
```

Comment:
I’ve moved this image code out to `FaviconHelper` as it was also needed for the password details screen